i18n/en.json

```diff
@@ -2567,6 +2567,7 @@
   "suggestion.mention.channels": "My Channels",
   "suggestion.mention.here": "Notifies everyone in the channel and online",
   "suggestion.mention.members": "Channel Members",
+  "suggestion.mention.moremembers": "Other Members",
```

Comment:
Should this be "Other Users"? Without "Channel" or "Team" for context, "Members" looks weird to be

Reply:
Where/when does this string occur?

I feel it was discussed, but I forget now, and didn't spot it on the latest spinmint below.

Reply:
This would appear over the loading section, but disappears once everything is merged together.

![image](https://user-images.githubusercontent.com/1023171/48878651-5d5d9b00-edd6-11e8-89b3-d7beece3d09e.png)

Reply:
Got it. I think `Members` referring to channel members is quite clear in this context.

Reply:
Are those users channel members though? I thought this was for loading people who aren't in the channel

Reply:
@hmhealey Some are channel members, some are not.

We are first loading users locally, so the query returns immediate results. We then load everyone else, most of which are not in the channel but a few that could be.

Once the loading has finished, there will be three sections: Channel Members, Special Mentions, and Not In Channel